addon/components/base/bs-dropdown.js

```diff
@@ -24,18 +24,17 @@ import layout from 'ember-bootstrap/templates/components/bs-dropdown';
  * `closeDropdown`
 
  ```hbs
- {{#bs-dropdown as |dd|}}
-   {{#dd.toggle}}Dropdown <span class="caret"></span>{{/dd.toggle}}
-   {{#dd.menu as |ddm|}}
-     {{#ddm.item}}
-       {{#ddm.link-to "index"}}Something{{/ddm.link-to}}
-     {{/ddm.item}}
-     {{ddm.divider}}
-     {{#ddm.item}}
-       {{#ddm.link-to "index"}}Something different{{/ddm.link-to}}
-     {{/ddm.item}}
-   {{/dd.menu}}
- {{/bs-dropdown}}
+  <BsDropdown as |dd|>
```

Comment:
Should be `DD`. At least for consistency, but I am not sure if it actually works without a starting capital letter, as this is a requirement for angle bracket invocation (which is happening here also, as `dd.toggle` is a component invocation). Again not totally sure...

Reply:
I've been using angle brackets, it works both ways, but I agree +1 for consistency

Reply:
but ... what about |ddm|? I think this might be hard to read like DD and DdM or something

Reply:
@simonihmig There is not requirement for variables to start with a capital letter. Even if they refer to an object which property is used as a component with angle bracket invocation: https://github.com/emberjs/rfcs/blob/master/text/0311-angle-bracket-invocation.md#dynamic-invocations
Fixed a wrong linting rule for that one a while ago: https://github.com/ember-template-lint/ember-template-lint/pull/474
I would prefer to keep them lowercase for readability but doesn't have a strong opinion about it.

Reply:
Ok, I am fine with either one. Just let's keep it consistent!

Reply:
[Guides](https://guides.emberjs.com/release/components/wrapping-content-in-a-component/) and [tutorial](https://guides.emberjs.com/release/tutorial/simple-component/) are both using lowercase for yielded variable.